public/main.js.LICENSE.txt

```diff
@@ -1,29 +0,0 @@
-/**
```

Comment:
Removed this file since it was supposed to be excluded by gitignore.  
We generally like to avoid commiting autogenerated files as long as they can always be recreated. Having them committed makes them show up with changes in pull requests everytime it changes which adds unnecessary noise to PRs.